### DISCUSSION TITLE
My solutions on `Compatibility with curl` in ubuntu 24.04

### DISCUSSION THREAD

**Discussion by AtticusZeller** [abraunegg/onedrive#3037] [Show and tell]
```bash
brew install curl
ls /home/linuxbrew/.linuxbrew/lib/libcurl*
# get things like /etc/ld.so.conf.d/linuxbrew-curl.conf.*
echo "/home/linuxbrew/.linuxbrew/lib" | sudo tee /etc/ld.so.conf.d/linuxbrew-curl.conf
# update
sudo ldconfig
```
now https2 works perfect!
here is my [configs](https://notes.atticuszeller.club/CS/OS/Ubuntu/System/#onedrive)

**Comment by abraunegg** [comment]
@AtticusZeller 
A few important notes regarding your configuration advice:

* Do not use `sync_dir = "~"` .... this means the client will *always* be scanning your home *entire* home directory for changes to upload to Microsoft OneDrive.
  * By using this approach you are exposing yourself to the following risks:
    * Sensitive Data Exposure
    * Unnecessary Data Synchronisation
    * Data Management Complexity
  * Please read https://github.com/abraunegg/onedrive/discussions/2992#discussioncomment-11429889 where this is expanded on
  * Your value `threads = "64"` is wrong. Please read the documentation
  * Do not use `--resync` unless the client specifically tells you to use it

**Reply by AtticusZeller** [reply]
Thank your for generous suggestions! 
I have remove --resync everytime from my notes and fix the num of threads

And make sure there no sensitive data from my home, just documents 

🤗
